mlrun/config.py

```diff
@@ -131,8 +131,9 @@
     "ignored_notebook_tags": "",
     # when set it will force the local=True in run_function(), set to "auto" will run local if there is no k8s
     "force_run_local": "auto",
-    # when set it will force the mock=True in deploy_function(), TBD allow "auto" to detect Nuclio
-    "mock_nuclio_deployment": "",
+    # when set it will force the mock=True in deploy_function(),
+    # set to "auto" will use mock of Nuclio is not detected (no nuclio_version)
+    "mock_nuclio_deployment": "auto",
```

Comment:
Add what are the supported options.
```suggestion
    # when set it will force the mock=True in deploy_function(),
    # set to "auto" will use mock of Nuclio if not detected (no nuclio_version)
    "mock_nuclio_deployment": "auto",
```